frameworks/kafka/tests/test_jmx.py

```diff
@@ -0,0 +1,204 @@
+
+    sdk_cmd.run_cli(
+        "security secrets create -f /tmp/{}-self-signed-keystore.ks /test/integration/kafka/keystore".format(
+            test_run))
+    sdk_cmd.run_cli(
+        "security secrets create -f /tmp/{}-passwordfile /test/integration/kafka/passwordfile".format(test_run))
+    sdk_cmd.run_cli(
+        "security secrets create -f /tmp/{}-keystorepass /test/integration/kafka/keypass".format(test_run))
+    sdk_cmd.run_cli("security secrets create -f /tmp/{}-access /test/integration/kafka/access".format(test_run))
+
+    service_options = {"service":
+                       {"name": foldered_name, "jmx":
+                        {"enabled": True, "port": 31199, "rmi_port": 31198,
+                         "password_file": "/test/integration/kafka/passwordfile",
+                         "access_file": "/test/integration/kafka/access",
+                         "key_store": "/test/integration/kafka/keystore",
+                         "key_store_password_file": "/test/integration/kafka/keypass"
+                         }
+                        }, "brokers": {"cpus": 0.5, "count": 1}}
+
+    if self_signed_trust_store:
+        service_options = sdk_utils.merge_dictionaries(
+            {"service": {"jmx": {"add_trust_store": True,
+                                 "trust_store": "/test/integration/kafka/keystore",
+                                 "trust_store_password_file": "/test/integration/kafka/keypass"}}},
+            service_options)
+    if sdk_utils.is_strict_mode():
+        kafka_sa = "test/integration/kafka/service-account"
+        kafka_sa_secret = "test/integration/kafka/secret"
+
+        sdk_security.setup_security(
+            config.SERVICE_NAME,
+            service_account=kafka_sa,
+            service_account_secret=kafka_sa_secret,
+        )
+        service_options = sdk_utils.merge_dictionaries(
+            {"service": {"service_account": kafka_sa, "service_account_secret": kafka_sa_secret}},
+            service_options
+        )
+
+    sdk_install.install(
+        package_name=config.PACKAGE_NAME,
+        service_name=foldered_name,
+        additional_options=service_options,
+        expected_running_tasks=1
+    )
+
+
+def uninstall_jmx_secrets():
+    sdk_security.delete_secret("/test/integration/kafka/keystore")
+    sdk_security.delete_secret("/test/integration/kafka/passwordfile")
+    sdk_security.delete_secret("/test/integration/kafka/access")
+    sdk_security.delete_secret("/test/integration/kafka/keypass")
+
+
+@pytest.mark.sanity
+def test_secure_jmx_configuration():
+    foldered_name = sdk_utils.get_foldered_name(config.SERVICE_NAME)
+
+    try:
+        install_jmx_configured_kafka()
+        broker_task_id_0 = sdk_tasks.get_task_ids(foldered_name)[0]
+        install_jmxterm(task_id=broker_task_id_0)
+        generate_jmx_command_files(task_id=broker_task_id_0)
+
+        cmd = "export JAVA_HOME=$(ls -d ${MESOS_SANDBOX}/jdk*/jre) && " \
```

Comment:
just note that when we merge openjdk 8 fix to all FWs, this path changes from `/jdk*/jre` to `/jdk*/`

Reply:
I think once we prepare the PR of bumping to openjdk8 it should pass all the tests and change it there